conans/test/unittests/client/conf/detect/__init__.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
```

Comment:
Hi @SSE4 

Please try to avoid adding this ``vim``-only line to all the files.